packages/ui/dialogs/transfer-eth/index.tsx

```diff
@@ -114,25 +114,24 @@ class TransferEth extends Component<Props, State> {
   }
 
   getLimit() {
-    const { formInputError } = this.state
     const { ethExchange, ethSentPastWeek, primaryCurrency } = this.props
     const remaining = String(Number(transferLimits(primaryCurrency)) - Number(ethSentPastWeek) * Number(ethExchange(primaryCurrency)))
     const end = remaining.indexOf('.') === -1 ? remaining.length : remaining.indexOf('.') + 3
     return remaining.slice(0, end)
   }
 
-  toFiat(amount, exchange, primaryCurrency) {
+  toFiat(amount, exchange) {
     if (amount === undefined) {
```

Comment:
Right now, if I type in simply `.` for the amount, the displayed fiat amount is `$NaN`
To fix that, consider something like this here:
`   
 if ( (amount === undefined) || (amount === '.') || (amount === ',') ) {
`
Or better would be a check against whatever the "decimal symbol" is...